docker/raiden.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Name=Raiden
+Version=XXVERSIONXX
+Exec=raiden
+Icon=raiden
```

Comment:
As far as I understand AppImage specification wants an icon. The file is just an svg (so no blob). So I kindly ask you to disregard that comment :P